Modules/Messages/test/Messages.Domain.Tests/Relationships/RelationshipTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Reflection;
+using Backbone.BuildingBlocks.Domain;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Messages.Domain.Entities;
+using Backbone.Modules.Messages.Domain.Ids;
+using Backbone.UnitTestTools.Data;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Messages.Domain.Tests.Relationships;
+public class RelationshipTests
+{
+    [Fact]
+    public void Relationship_must_be_active_to_allow_sending_messages()
```

Comment:
Add another test to make sure it's not allowed to send messages in status `Terminated`.

Reply:
I did this:
```c#
    [Fact]
    public void Relationship_cannot_be_terminated_to_allow_sending_messages()
    {
        // Arrange
        var relationship = CreateRelationship(RelationshipStatus.Terminated);

        // Act
        var acting = () => relationship.EnsureSendingMessagesIsAllowed(0, 5);

        // Assert
        acting.Should().Throw<DomainException>().Which.Code.Should().Be("error.platform.validation.message.relationshipToRecipientNotActive");
    }
```

It is very similar to the first test in the file though. Maybe I misunderstood you.

Changes made.

Reply:
Yeah. Just one minor comment regarding the phrasing: maybe the tests should be called "Sending_messages_is_not_allowed_when_relationship_is_*"

Reply:
Ahm, and correspondingly "Sending_messages_is_allowed_when_relationship_is_active"